person.rb

```diff
@@ -16,5 +18,9 @@ def of_age?
   def can_use_services?
     @parent_permision || of_age?
   end
+
+  def validate_name
+    @name = @corrector.correct_name(@name)
+  end
   private :of_age?
```

Comment:
- [ ] You should create a `Decorator` class that inherits from Namable and implement the `correct_name ` method that returns the result of the `correct_name ` method of the @namable.
- [ ] You should create a `CapitalizeDecorator`  class and `TrimmerDecorator` class that inherits from `Decorator`